JavaScript/5.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Task 5</title>
+</head>
+<body>
+	<span id="result"></span>
+	<script>
+		var primeFactors = function (n) {
+			var i = 2;
+    		var factors = [];
+    		while (i * i <= n){
+		        if (n % i != 0) {
+		            i += 1;
+		        } else {
+			        n /= i;
+			        factors.push(i);
+			    };
+			};
+		    if (n > 1) {
+		        factors.push(n);
+		    };
+		    return factors;
+		};
+
+		var countElements = function (array, element) {
+			var count = 0;
+			for (var k = 0; k <= array.length; k++) {
```

Comment:
Array has array.length elements from `array[0]` till `array[array.length - 1]`
`for (var k = 0; k <= array.length - 1; k++) `